### ISSUE TITLE
Paid Version

### ISSUE THREAD

**Issue by scottkuhlman** [TinyCrayon/TinyCrayon-iOS-SDK#21]
We would also like a paid version so we can eliminate the Banner Ad.  Please let us know the cost/price for this.  Thank you

**Comment by terraoldham** [comment]
Would also love to see a paid version to remove the banner ad. Would this be possible? 

Thanks!

CC: @bblia

**Comment by bblia** [comment]
yes please! ^^^^

**Comment by davidroman0203** [comment]
I'd like a paid version to remove the banner ad, too. Where else should I contact for it?
Thanks.

**Comment by Boca007** [comment]
yes please. me too. im interested about it.

**Comment by codenia** [comment]
I also have an interest in a paid version without advertising.

**Comment by DevSoFar** [comment]
Me too

**Comment by forsan** [comment]
i can pay up to $5000 for paid version.

**Comment by alexsorokoletov** [comment]
@bingh0616 any news on the updated version? 
The email address `service@tinycrayon.com` does not work.

**Comment by bingh0616** [comment]
Hi @alexsorokoletov , we should have the updated version by end of this month.

**Comment by scottkuhlman** [comment]
Are you considering a PAID version?  We would like to get rid of the Banner Ad.  We will pay you for this.

Scott Kuhlman

On Mar 6, 2019, at 11:55 PM, Bing Hu <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi @alexsorokoletov<https://github.com/alexsorokoletov> , we should have the updated version by end of this month.

**Comment by bingh0616** [comment]
Yes @scottkuhlman , we plan to have a version you can pay to get rid of the Ad.

**Comment by scottkuhlman** [comment]
When do you think this will be available?

Scott Kuhlman

On Mar 7, 2019, at 1:23 AM, Bing Hu <notifications@github.com<mailto:notifications@github.com>> wrote:


Yes @scottkuhlman<https://github.com/scottkuhlman> , we plan to have a version you can pay to get rid of the Ad.

**Comment by bingh0616** [comment]
Hopefully end of this month, we are on it.

**Comment by bingh0616** [comment]
@adrianorezena , we are testing the code now, should be ready soon.

**Comment by keenen** [comment]
@bingh0616 would love to know if there's any update on this! We're waiting for this as well.

**Comment by LeonardRockstar** [comment]
@bingh0616 We're eagerly waiting for the paid version too

**Comment by scottkuhlman** [comment]
Have you changed the code?  We have used your old version and it works brilliantly.  We have built a new APP and it is not working.   We have sent emails to your service@tinycrayon.com however no response.  Can you please update us on everything.  Thanks

**Comment by scottkuhlman** [comment]
@bingh0616  can you please review and give us an update.   We eagerly await the updated version.  In the meantime have you chanced anything that we need to know about.

**Comment by LeonardRockstar** [comment]
@bingh0616 We'd really appreciated an update on the progress too. We'd love to pay for an updated version of TinyCrayon, but we're going to have to find an alternative if the new version isn't coming anytime soon.

**Comment by GGustavoLopez** [comment]
@bingh0616 We are highly interested to hear any updates of the paid version or if there is any possibility to get an early access even it it steels in beta we would love it!.

**Comment by AdamWorld** [comment]
Please, this is awesome sdk. when can i get paid version?

**Comment by yousaf94** [comment]
Hi
If you have the TinyCrayon Library available with you, Can you please share with us ?
